### ISSUE TITLE
Changing octave?

### ISSUE THREAD

**Issue by ericandrewlewis** [mmontag/dx7-synth-js#7]
Thanks for this!

Is it possible to change the octave? Can't get low enough for the Bizarre Love Triangle bass line.

**Comment by mmontag** [comment]
No, but you could divide the Coarse pitch of all the operators by 2 in a pinch :)
